cli/endpoints/online/mlflow-vnet/sklearn-deployment.yaml

```diff
@@ -0,0 +1,12 @@
+$schema: https://azuremlschemas.azureedge.net/latest/managedOnlineDeployment.schema.json
+name: sklearn-deployment
+endpoint_name: my-endpoint
+model:
+  name: mlflow-model-vnet
+  version: 1
+  local_path: model-vnet
+environment_variables:
+  MLFLOW_MODEL_FOLDER: model-vnet
+environment: azureml:vnet-400:1
```

Comment:
@shivanissambare - this line has to pick up the environment created in shell script